### PR TITLE
Add scheduled reminder limitiations.

### PR DESCRIPTION
**Pull request by tischenko-2-12**
### Why:

**Closes [issue link]**

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ versions:
 
 Team maintainers and organization owners can set scheduled reminders for any pull requests that a team has been requested to review. Before you can create a scheduled reminder for your team, an organization owner must authorize your Slack workspace. For more information, see "[Managing scheduled reminders for your organization](/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-organization)."
 
+{% note %}
+
```
Comment (18mkt) [inline content/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team.md:17]: #4343

File: content/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team.md [hunk 2]
```diff
@@ -13,6 +13,14 @@ versions:
...
+**Note**: There are some limitations when setting up scheduled reminders:
+- At an organization level, there's a limit of five repositories.
+- At a repository level, you will only receive notifications for the oldest 20 pull requests.
+
+{% endnote %}
+
 ### Creating a scheduled reminder for a team
 {% data reusables.profile.access_profile %}
```
Comment (18mkt) [inline content/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team.md:25]: #2709

**Review by 18mkt — CHANGES_REQUESTED** [review]
#4343 #2709 #4326 #4342

**Comment by 18mkt** [comment]
> Thanks for opening this pull request! A GitHub docs team member should be by to give feedback soon. In the meantime, please check out the [contributing guidelines](https://github.com/github/docs/blob/main/CONTRIBUTING.md).

**Review by 18mkt — CHANGES_REQUESTED** [review]
#4343 #2709

**Comment by 18mkt** [comment]
#4343

**Comment by 18mkt** [comment]
#2709

**Comment by 18mkt** [comment]
I'm

**Review by clubiriesouthbaya — APPROVED** [review]
> 

>

**Pull request closed without merging**